renderer/src/components/TotalJobsCompleted.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect, useState } from 'react'
+
+export const TotalJobsCompleted : React.FC = () => {
+  const [totalJobCount, setTotalJobCount] = useState<number>(0)
+
+  useEffect(() => {
+    (async () => {
+      const count = await window.electron.getTotalJobsCompleted()
+      setTotalJobCount(count)
+    })()
+  }, [])
+
+  useEffect(() => {
+    const unsubscribe = window.electron.onJobStatsUpdated(count => {
+      setTotalJobCount(count)
+    })
+    return unsubscribe
+  }, [])
+
+  // TODO: add proper CSS styling :)
+  const style: React.CSSProperties = {
+    marginLeft: '2em',
+    fontSize: '2em',
+    fontFamily: 'monospace'
+  }
+
+  return <>
+    <h3>Total jobs completed</h3>
+    <p style={style}>{totalJobCount}</p>
```

Comment:
```suggestion
  const [totalJobsCompleted setTotalJobsCompleted] = useState<number>(0)

  useEffect(() => {
    (async () => {
      const jobsCompleted = await window.electron.getTotalJobsCompleted()
      setTotalJobsCompleted(jobsCompleted)
    })()
  }, [])

  useEffect(() => {
    const unsubscribe = window.electron.onJobStatsUpdated(count => {
      setTotalJobsCompleted(count)
    })
    return unsubscribe
  }, [])

  // TODO: add proper CSS styling :)
  const style: React.CSSProperties = {
    marginLeft: '2em',
    fontSize: '2em',
    fontFamily: 'monospace'
  }

  return <>
    <h3>Total jobs completed</h3>
    <p style={style}>{totalJobsCompleted}</p>
```

I find consistent naming really helpful

Reply:
Thank you! I missed this part; I agree it's much better renamed.